iroh-net/src/endpoint.rs

```diff
@@ -59,6 +59,9 @@ pub use iroh_base::node_addr::{AddrInfo, NodeAddr};
 /// is still no connection the configured [`Discovery`] will be used however.
 const DISCOVERY_WAIT_PERIOD: Duration = Duration::from_millis(500);
 
+/// Environment variable to force the use of staging relays.
+pub const ENV_FORCE_STAGING_RELAYS: &str = "IROH_FORCE_STAGING_RELAYS";
```

Comment:
if you put the env var in here, I would also put the check in here and make it public function

Reply:
does this still need `pub`?